app/models/manageiq/providers/vmware/cloud_manager.rb

```diff
@@ -0,0 +1,96 @@
+class ManageIQ::Providers::Vmware::CloudManager < ManageIQ::Providers::CloudManager
+  require_nested :RefreshParser
+  require_nested :RefreshWorker
+  require_nested :Refresher
+  require_nested :Vm
+
+  def self.ems_type
+    @ems_type ||= "vmware_vcd".freeze
```

Comment:
Should the ems_type be `"vmware_cloud"` since OpenStack's `InfraManager` is `openstack_infra`?  @Fryguy do you have a preference?